addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -140,8 +140,7 @@ const Link = Widget.extend({
         if (this.data.url) {
             var match = /mailto:(.+)/.exec(this.data.url);
             this.$('input[name="url"]').val(match ? match[1] : this.data.url);
-            this._onURLInput();
-            this._savedURLInputOnDestroy = false;
+            this._adaptForm();
```

Comment:
As explained on the task, this line was _not_ the problem. The problem is in `_onURLInput` so removing this call won't fix it. It just removes a case where the function is called but whenever the function is called you still have the problem.

Reply:
Look it's ok I'm taking care of it.

Reply:
@bso-odoo @dmo-odoo Here's what I suggest instead: https://github.com/odoo/odoo/pull/108281 (and https://github.com/odoo/odoo/pull/108269 for master)